bentoml/frameworks/sklearn.py

```diff
@@ -33,7 +33,7 @@ class SklearnModelArtifact(BentoServiceArtifact):
         pickle_extension (str): The extension format for pickled file
 
     Raises:
-        MissingDependencyException: sklean package is required for SklearnModelArtifact
+        MissingDependencyException: sklearn package is required for SklearnModelArtifact
```

Comment:
The minor docs changes in this file are unrelated to adding EvalML support. I noticed the typo and missing command and wanted to add a fix.

Reply:
Thank you!